Azure/sonic-buildimage
# Added Netberg Aurora 715 support

#### Why I did it

Suport Innovium based platform Netberg Aurora 715
ASIC: Innovium Teralynx IVM 55300
Pors: 32x 100G

#### How I did it

Added specification x86_64-netberg_aurora_715-r0 device/netberg directory
Added platform/innovium/sonic-platform-modules-netberg contains kernel modules, scripts and sonic_platform package. 
Modified the platform/innovium/one-image.mk and platform/innovium/rule.mk to include Aurora 715 related ID and files. 

#### How to verify it
Build sonic
Install the image on the device and verify the related components are installed and shown correctly. 

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog

Initial support for Netberg Aurora 715

#### A picture of a cute animal (not mandatory but encouraged)
